app/controllers/publishers/omniauth_callbacks_controller.rb

```diff
@@ -297,5 +297,9 @@ def require_publisher_not_created_through_youtube_auth
         redirect_to(home_publishers_path)
       end
     end
+
+    def redirect_options
```

Comment:
verify this dynamically

Reply:
removed- it was a mistake that this change was included initially.